package.json

```diff
@@ -0,0 +1,81 @@
+{
+  "name": "ember-brand-manager",
```

Comment:
can you rename it to `@upfluence/ember-brand-manager`. Otherwise it won't be deployable on the Github registry